deltachat-ffi/deltachat.h

```diff
@@ -1011,6 +1011,7 @@ dc_msg_t*       dc_get_draft                 (dc_context_t* context, uint32_t ch
 
 
 #define         DC_GCM_ADDDAYMARKER          0x01
+#define         DC_GCM_SYSTEM_ONLY           0x02
```

Comment:
the flag corresponds to the api `dc_msg_is_info()` - maybe better call it  `DC_GCM_INFO_ONLY` therefore? same for the corresponding rust- and python-defines.